charts/sourcegraph/templates/prometheus/prometheus.ConfigMap.yaml

```diff
@@ -249,7 +254,7 @@ data:
         target_label: ns
       # Sourcegraph specific customization. We want to add a label to every 
       # metric that indicates the node it came from.
-      - source_labels: [__meta_kubernetes_endpoint_node_name]
+      - source_labels: [__meta_kubernetes_pod_node_name]
```

Comment:
this seems unrelated? what was the reason for this change?

Reply:
This was just a typo that existed in the original version of the prometheus node-exporter changes that I silently fixed here. `__meta_kubernetes_pod_node_name` is the correct name for this label (since we're scraping pods, not endpoints).  I figured this wasn't too bad since nobody has deployed this helm chart yet.